autopush/db.py

```diff
@@ -49,6 +49,8 @@ def hasher(uaid):
 
 
 def normalize_id(id):
+    if id is None:
+        return None
```

Comment:
There was no test for this before, so I had removed it. Is there any case where this will be called on a None or empty string?